.github/workflows/ci-clang-format.yml

```diff
@@ -72,7 +72,7 @@ jobs:
     - name: Create Build Environment
       run: |
         sudo apt-get -y install doxygen vera++ zlib1g-dev libsnappy-dev \
-          liblz4-dev clang-format-9 libunwind-dev
+          liblz4-dev clang-format-12 libunwind-dev
```

Comment:
Mention this upgrade in the PR's description too?

Reply:
+1, will do.  I considered doing a separate PR.  One of the files changed here didn't have the function return type on its own line in clang-format-9 and my laptop has 14 and can't easily install 9 so it seemed best to just upgrade.